client/lib/signup/test/flow-controller.js

```diff
@@ -85,7 +67,7 @@ describe( 'flow-controller', () => {
 				reduxStore: store,
 			} );
 
-			expect( getSignupProgress( store.getState() ) ).toHaveLength( 0 );
+			expect( isEmpty( getSignupProgress( store.getState() ) ) ).toBe( true );
```

Comment:
This is just a tiny style nit, but It looks like a common pattern in other tests in the codebase for checking for an empty object is to use Jest's `.toEqual( {} );` so the lodash `isEmpty` might be unnecessary here and on line 86.